app/views/dispatch/forecast.erb

```diff
@@ -0,0 +1,65 @@
+<div class="container">
+
+	<div class='row'>
+		<div class='col-xs-12'>
+			<div class='panel panel-default'>
+				<div class='panel-heading'>
+					Active Missions
+				</div>
+				<% if @active_missions.length == 0 %>
```

Comment:
too much logic in the view here. create a helper that does this comparison for you!